extensions-core/kafka-indexing-service/src/test/java/org/apache/druid/indexing/kafka/KafkaIndexTaskTest.java

```diff
@@ -905,12 +905,13 @@ public void testIncrementalHandOffReadsThroughEndOffsets() throws Exception
 
     final ListenableFuture<TaskStatus> normalReplicaFuture = runTask(normalReplica);
     // Simulating one replica is slower than the other
-    final ListenableFuture<TaskStatus> staleReplicaFuture = Futures.transform(
+    final ListenableFuture<TaskStatus> staleReplicaFuture = Futures.transformAsync(
         taskExec.submit(() -> {
           Thread.sleep(1000);
           return staleReplica;
         }),
-        (AsyncFunction<Task, TaskStatus>) this::runTask
+        (AsyncFunction<Task, TaskStatus>) this::runTask,
+        Execs.directExecutor()
```

Comment:
Why it is `Execs.directExecutor()`?

Reply:
@wangyum
Because upgrading Guava requires an explicit Executor passed to the method.  The previous behavior was to implicitly pass `directExecutor` if no Executor was provided.  I tried to pass the guava version and some maven build tool plugin rejected the change and directed me to use this instead.